docs/roadmap.md

```diff
@@ -0,0 +1,11 @@
+# Roadmap
+
+Welcome to the roadmap for Melange! While Melange is now at version 1.0, our team is actively developing and improving it to create the best OCaml experience on JavaScript.
```

Comment:
Maybe this section can start just with the sentence below ("This page provides...")